2.0.0/arm64/entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -x
+set -euo pipefail
+IFS=$'\n\t'
+
+# Add openhab user & handle possible device groups for different host systems
+# Container base image puts dialout on group id 20, uucp on id 10
+# GPIO Group for RPI access
+NEW_USER_ID=${USER_ID:-9001}
+echo "Starting with UID : $NEW_USER_ID"
+adduser -u $NEW_USER_ID --disabled-password --gecos '' --home ${APPDIR} openhab &&\
```

Comment:
The entrypoint.sh is called each time the container is started, so even if it would work to run adduser each time, it has a bad smell.

It would be better to create the user by using the Dockerfile at container image creation time. the chown on the other hand is fine as that might change by mounting volumes inside the container.